### PR TITLE
Add qcon alias in torq.sh script

### PR DESCRIPTION
**Pull request by FionaMorgan**
# qcon alias! :computer:

### PR general description
Adding qcon flag such that `./torq.sh qcon <processname> <username>:<password>` will connect to the process without need for port to be entered manually. Fixing the number of arguments allowed as input for the debug flag. Neaten final flag case statements by wrapping each case in separate function.

### Final Checklist
- [x] My code follows the code style of this project
- [x] My code is commented appropriately and concisely
- [x] If necessary, I have added new appropriate labels to this PR
- [x] I have assigned the appropriate users to this PR

### CODE REVIEW

File: torq.sh [hunk 1]
```diff
@@ -190,7 +205,7 @@ getextrascsv() {
 
 checkextrascsv() {
   if [[ $(echo ${BASH_ARGV[*]} | grep -e extras) ]] && [[ $(echo ${BASH_ARGV[*]} | grep -e csv) ]]; then
-    getextrascsv $@;                                                                                # gets extras and csv arguments 
+    getextrascsv $@;
```
Comment (ThomasSmyth) [inline torq.sh]: Why remove the comment?

File: torq.sh [hunk 2]
```diff
@@ -206,23 +221,78 @@ allcsv() {
  }
 
 startprocs() {
+  checkextrascsv "$*";
```
Comment (ThomasSmyth) [inline torq.sh]: Comment for this? (and line 231)

File: torq.sh [hunk 3]
```diff
@@ -206,23 +221,78 @@ allcsv() {
...
+runprint() {
+  checkextrascsv "$*";
+  for p in $PROCS; do
+    print "$p";
+  done
+ }
+
+rundebug() {
+  checkextrascsv "$*";
+  if [[ $(echo $PROCS | wc -w) -gt 1 ]]||[[ $# -ne 2 ]]; then
```
Comment (ThomasSmyth) [inline torq.sh]: Drop some spaces in there for clarity:
```
... ]] || [[ ...
```

File: setenv.sh [hunk 4]
```diff
@@ -27,3 +27,6 @@ export TORQPROCESSES=${KDBAPPCONFIG}/process.csv
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$KDBLIB/l32
 
 export TORQMONIT=${TORQHOME}/logs/monit                                                             # set the folder for monit outputs
+
+export RLWRAP=QCON=/usr/bin/rlwrap                                                                  # set rlwrap path
```
Comment (ThomasSmyth) [inline setenv.sh]: Won't this set `RLWRAP` to `QCON=/usr/bin/rlwrap`, is that right?

Comment (FionaMorgan) [reply setenv.sh]: This is a typo, Fixed it!

**Pull request merged**